deps/wazuh_testing/wazuh_testing/scripts/simulate_agents.py

```diff
@@ -0,0 +1,157 @@
+import argparse
+import logging
+import os
+from multiprocessing import Process
+from time import sleep
+
+import wazuh_testing.tools.agent_simulator as ag
+from wazuh_testing import TCP
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+def run_agents(agents_number=1, manager_address='localhost', protocol=TCP, agent_version='v4.0.0',
+               agent_os='debian8', eps=1000, run_duration=20, active_modules=[], modules_eps=None):
+    """Run a batch of agents connected to a manager with the same parameters.
+
+    Args:
+        agents_number (int): Number of agents to run.
+        manager_address (str): Manager address (hostname or IP).
+        protocol (str): Communication protocol.
+        agent_version (str): Agents version p.e: v4.0.0
+        agent_os (str): Agents os, some examples: debian8, ubuntu18.04, mojave...
+        eps (int): Total events per second sent by each agent to the manager.
+        run_duration (int): Agent life time.
+        active_modules (list): list with active modules names.
+        modules_eps (list): list with eps for each active modules.
+    """
+
+    logger = logging.getLogger(f"P{os.getpid()}")
+    logger.info(f"Starting {agents_number} agents.")
+
+    active_agents, injectors = [], []
+
+
+    for _ in range(agents_number):
+        agent = ag.Agent(manager_address, "aes", os=agent_os, version=agent_version, fim_eps=eps)
+        available_modules = agent.modules.keys()
+        sending_modules = len(active_modules)
+        if 'receive_messages' in active_modules:
+            sending_modules -= 1
+
+        for module in active_modules:
+            if module not in available_modules:
+                raise ValueError(f"Selected module: '{module}' doesn't exist on agent simulator!")
+            
+        for index, module in enumerate(available_modules):
+            if module in active_modules:
+                agent.modules[module]['status'] = 'enabled'
+                
+                if modules_eps is not None and 'eps' in agent.modules[module]:
+                    agent.modules[module]['eps'] = modules_eps[index]
+                else:
+                    agent.modules[module]['eps'] = eps/sending_modules
```

Comment:
It is better if the result is integer

Reply:
https://github.com/wazuh/wazuh-qa/pull/1148/commits/57e715ec0b50b5962b693361f56d3df383006a55